Modules/Loadable/Segmentations/Logic/vtkImageLabelDilate3D.h

```diff
@@ -0,0 +1,74 @@
+/*=========================================================================
+
+  Program:   Visualization Toolkit
+  Module:    vtkImageLabelDilate3D.h
+
+  Copyright (c) Ken Martin, Will Schroeder, Bill Lorensen
+  All rights reserved.
+  See Copyright.txt or http://www.kitware.com/Copyright.htm for details.
+
+     This software is distributed WITHOUT ANY WARRANTY; without even
+     the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+     PURPOSE.  See the above copyright notice for more information.
+
+=========================================================================*/
+/**
+ * @class   vtkImageLabelDilate3D
+ * @brief   Median Filter
+ *
+ * vtkImageLabelDilate3D dilates a labelmap by replacing each background voxel with the
+ * most dominant label voxel in its neighborhood.
+ */
+
+#ifndef vtkImageLabelDilate3D_h
+#define vtkImageLabelDilate3D_h
+
+#include "vtkImageSpatialAlgorithm.h"
+#include "vtkSlicerSegmentationsModuleLogicExport.h"
+
+class VTK_SLICER_SEGMENTATIONS_LOGIC_EXPORT vtkImageLabelDilate3D : public vtkImageSpatialAlgorithm
+{
+public:
+  static vtkImageLabelDilate3D* New();
+  vtkTypeMacro(vtkImageLabelDilate3D, vtkImageSpatialAlgorithm);
+  void PrintSelf(ostream& os, vtkIndent indent) override;
+
+  /**
+   * This method sets the size of the neighborhood.  It also sets the
+   * default middle of the neighborhood
+   */
+  void SetKernelSize(int size0, int size1, int size2);
+
+  ///@{
+  /**
+   * Return the number of elements in the median mask
+   */
+  vtkGetMacro(NumberOfElements, int);
+  ///@}
```

Comment:
Not really clear what this refers to without reading the code. Maybe some extra text about why one would want to know this could help.

Reply:
This accessor was there in the median filter that this filter was adapted from. The line was added by Bill Lorensen in 1998 when he changed the `NumberOfElements` member variable from public to protected. While it does not do any harm, it is just noise, I'll remove it.